src/audioprofile/AudioProfile.ts

```diff
@@ -32,4 +32,18 @@ export default class AudioProfile {
   static fullbandMusicMono(): AudioProfile {
     return new AudioProfile(64000);
   }
+
+  /**
+   * Creates an AudioProfile optimized for fullband stereo music (128 kbit/s stereo).
+   */
+  static fullbandMusicStereo(): AudioProfile {
+    return new AudioProfile(128000);
+  }
+
+  /**
+   * Returns true if audio profile is set to stereo mode.
+   */
+  isStereo(): boolean {
+    return this.audioBitrateBps === AudioProfile.fullbandMusicStereo().audioBitrateBps;
```

Comment:
nit: Seem a bit roundabout to just compare it to 12800 right?

Reply:
Updated